server/main.lua

```diff
@@ -57,18 +57,25 @@ AddEventHandler('playerDropped', function()
 end)
 
 ---Lock every spawned vehicle
----@param vehicle number The entity number of the vehicle.
-AddEventHandler('entityCreated', function (vehicle)
-    if not vehicle
-        or type(vehicle) ~= 'number'
-        or not DoesEntityExist(vehicle)
-        or GetEntityPopulationType(vehicle) > 5
-        or GetEntityType(vehicle) ~= 2
+---@param entity number The entity number of the vehicle.
+AddEventHandler('entityCreated', function (entity)
+    if not entity
+        or type(entity) ~= 'number'
+        or not DoesEntityExist(entity)
+        or GetEntityPopulationType(entity) > 5
     then return end
-    local isDriver = GetPedInVehicleSeat(vehicle, -1) ~= 0
+
+    local type = GetEntityType(entity)
+
+    if type ~= 2 and type ~= 1 then
```

Comment:
I think the vehicle types could benefit from an enum

Reply:
done